packages/rich-text/src/to-dom.js

```diff
@@ -251,6 +251,15 @@ export function applyValue( future, current ) {
 	}
 }
 
+function isRangeEqual( a, b ) {
```

Comment:
Out of curiosity, is this something we could be using `Range#compareBoundaryPoints` for, testing a return value of `0`?

https://developer.mozilla.org/en-US/docs/Web/API/Range/compareBoundaryPoints

Reply:
I wonder how much of a difference this makes. We'd have to call the method twice with `Range.START_TO_START` and `Range.END_TO_END`. I'm fine doing it either way, no strong opinion.

Reply:
> I wonder how much of a difference this makes. We'd have to call the method twice with `Range.START_TO_START` and `Range.END_TO_END`. I'm fine doing it either way, no strong opinion.

Hm, in my mind it was just going to be one `START_TO_END` call. If not, let's avoid.

Reply:
> > I wonder how much of a difference this makes. We'd have to call the method twice with `Range.START_TO_START` and `Range.END_TO_END`. I'm fine doing it either way, no strong opinion.
> 
> Hm, in my mind it was just going to be one `START_TO_END` call. If not, let's avoid.

i.e.

```js
const isRangeEqual = ( a, b ) => a.compareBoundaryPoints( Range.START_TO_END, b ) === 0;
```

Reply:
I read `START_TO_END` as: compare the start of `a` to the end of `b`. There's the opposite as well, so what would the difference otherwise be? I'll test to be sure.

Reply:
Okay, this method compares the range's _visual_ boundary points, not the range properties. See  `Range.getBoundingClientRect()` as well.